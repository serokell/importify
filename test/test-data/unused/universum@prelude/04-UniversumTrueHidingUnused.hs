module UniversumTrueHidingUnused where

import           Prelude   (Bool (True))
import           Universum hiding (Bool (True), show)

foo :: Bool
foo = True
bar = undefined
