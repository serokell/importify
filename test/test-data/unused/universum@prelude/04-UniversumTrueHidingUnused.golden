module UniversumTrueHidingUnused where

import           Prelude   (Bool (True))
import           Universum hiding (Bool             )

foo :: Bool
foo = True
bar = undefined
